src/server/index.jsx

```diff
@@ -124,6 +125,17 @@ if (process.env.APP_ENV === 'local') {
 
       sendDataFile(res, `${dataFilePath}.json`, next);
     })
+    .get(mediaAssetPageDataRegexPath, async ({ params }, res, next) => {
```

Comment:
We should add a test for this similar to https://github.com/bbc/simorgh/blob/ec6eacd74ba438e1471783de58af2a02a79ff9fd/src/server/index.test.jsx#L485-L505

Reply:
If we don't have this test the code coverage drops for the server file and CodeClimate complains in the PR checks...

![image](https://user-images.githubusercontent.com/7791726/64604118-7b25dc00-d3b9-11e9-9a60-f8deb8f09f12.png)

Reply:
Thanks Phil